added nvm use

DaniRey opened the pull request:
To use the correct node version, the usage of `nvm use` is recommended - see https://github.com/unic/estatico/blob/develop/docs/Setup_and_Usage.md
Reflect this recommendation in the README.md

DaniRey committed:
added nvm use

To use the correct node version, the usage of `nvm use` is recommended - see https://github.com/unic/estatico/blob/develop/docs/Setup_and_Usage.md
Reflect this recommendation in the README.md

marbor3 commented:
I'm not sure we even need this part in `README.md` personally I would remove whole Getting started section - new users should really start with just `Setup_and_Usage.md`

What do you think? @DaniRey, @backflip @orioltf ?

orioltf commented:
I agree with @marbor3. @DaniRey what about if we link `Setup_and_Usage.md` in the `Getting started` section? This would simplify and reduce maintenance.

DaniRey commented:
That would be even better.

DaniRey committed:
Removed the getting started section

Users shall follow the link to https://github.com/unic/estatico/blob/develop/docs/Setup_and_Usage.md in the Documentation -> Setup / Usage chapter
